lessons/python/prerequisites/prerequisites-ts.md

```diff
@@ -0,0 +1,46 @@
+# Prerequisites (Python)
+
+## Frameworks & Tooling 🧰
+
+In order to complete the the lessons using Python you need to install the following:
+
+|Prerequisite|Lessons|Description
+|-|-|-
+|[Node.js](https://nodejs.org/en/download/)|All| The Node.js runtime including the node package manager NPM. Install an _LTS_ version.
+|[Python](https://www.python.org/downloads/)|All| Python releases. Install 3.9 version.
+|[VSCode](https://code.visualstudio.com/Download)|All|A great cross platform code editor.
+|[VSCode Azure Functions extension](https://github.com/Microsoft/vscode-azurefunctions)|All|Extension for VSCode to easily develop and manage Azure Functions.
+|[Azure Functions Core Tools](https://github.com/Azure/azure-functions-core-tools)|All|Azure Functions runtime and CLI for local development.
+|[RESTClient for VSCode](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) or [Postman](https://www.postman.com/)|All|An extension or  application to make HTTP requests.
+|[Azure CLI](https://docs.microsoft.com/en-us/cli/azure/what-is-azure-cli)|Deployment|Command line interface used to manage Azure resources. Can be run on your local dev environment, in a deployment pipeline or in the [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview).
+
+> 📝 **Tip** - Azure Functions only support the long term support (LTS) versions of Node.js as shown in the [Azure Functions runtime versions overview](https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions). Consequently, there is always a bit of a lag between the official availability of a Node.js LTS release and the official support of this release in Azure Functions.
```

Comment:
This section can be adjusted to mention Python versions. I suggest adding this link (https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python?tabs=azurecli-linux%2Capplication-level#python-version) so we point to the official docs instead of repeating content that will get outdated quickly